src/unsubscribe.js

```diff
@@ -1,14 +1,14 @@
 import { Provider } from './providers'
 
 /**
-   * @name unsubscribe
-   * @description Unsubscribes from listening to a component
-   * @param {Function} handler The handler function
-   **/
-  export default ( handler ) => {
-    Provider._handlers.forEach( ( fn, index ) => {
-      if ( fn === handler) {
-        Provider._handlers.splice(index, 1)
-      }
-    })
-  }
\ No newline at end of file
+ * @name unsubscribe
+ * @description Unsubscribes from listening to a component
+ * @param {Function} handler The handler function
+ **/
+export default (handler) => {
+  Provider._handlers.forEach((fn, index) => {
```

Comment:
Admittedly this is a matter of taste, but most code I've seen omits spaces after and before function argument parentheses, but keeps them in lists and objects.